region/brinstar/red/Beta Power Bomb Room.json

```diff
@@ -287,11 +301,7 @@
           "explicitWeapons": ["PowerBomb"]
         }}
       ],
-      "clearsObstacles": ["A", "B"],
-      "devNote": [
-        "This is essentially redundant with the Morph Power Beam kill.",
-        "It wouldn't be redundant if Power Beam were to be taken away, though."
-      ]
```

Comment:
This note doesn't seem useful to me.  Killing the hoppers with powerbeam is now far more difficult, and powerbeam doesn't break the blocks, which changes everything.

Reply:
Yeah. This type of comment isnt useful and can always be deleted